src/style.css

```diff
@@ -0,0 +1,144 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  height: 406px;
+  width: 200px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  --min-gap: 16px;
+}
+
+.product-card__computer-view {
+  margin: 32px 19px 40px;
```

Comment:
[STYLES] - Be consistent with your margins. Add only top or bottom, don't add both.